src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php

```diff
@@ -293,21 +291,17 @@ protected function sortServiceIds(array $serviceIds)
     public static function getClassDescription(string $class, string &$resolvedClass = null): string
     {
         $resolvedClass = $class;
-
-        if (!interface_exists(DocBlockFactoryInterface::class)) {
-            return '';
-        }
-
         try {
             $r = new \ReflectionClass($class);
-            $resolvedClass = $r->name;
-
-            if ($docComment = $r->getDocComment()) {
-                return DocBlockFactory::createInstance()
-                    ->create($docComment)
-                    ->getSummary();
-            }
-        } catch (\ReflectionException | \InvalidArgumentException $e) {
+            $docComment = $r->getDocComment();
+            $summary = str_replace(
+                array('/', '*', "\t", "\r\n", "\n", "\r", '  '),
+                '',
+                substr($docComment, 0, strpos($docComment, '@'))
```

Comment:
this seems very fragile (perhaps `* @`) works better. But tend to prefer an explicit regex to capture the short description as a sentence.

This captures both long+short without newlines

Reply:
Can you help me with a regex to get the description? Would be great!

Reply:
@SerkanYildiz something like https://regex101.com/r/Sqcmmf/1/

It matches the first sentence per https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#51-summary

> A Summary MUST end with two sequential line breaks, unless it is the only content in the PHPDoc.

The current regex does not include single new lines, e.g. `short line 1\nshort line 2` would only match `short line 1`. AFAIK it's OK.